Slapper/src/slapper/entities/SlapperHuman.php

```diff
@@ -27,7 +27,7 @@ public function spawnTo(Player $player)
             $pk->pitch = $this->pitch;
             $pk->item = $this->getInventory()->getItemInHand();
             $pk->metadata = [
-                self::DATA_FLAGS => [self::DATA_TYPE_LONG, ((1 << self::DATA_FLAG_NO_AI) | ($this->isNameTagVisible() ? (1 << self::DATA_FLAG_CAN_SHOW_NAMETAG) : 0))],
+                self::DATA_FLAGS => [self::DATA_TYPE_LONG, ((1 << self::DATA_FLAG_NO_AI) | (1 << self::DATA_FLAG_ALWAYS_SHOW_NAMETAG) | ($this->isNameTagVisible() ? (1 << self::DATA_FLAG_CAN_SHOW_NAMETAG) : 0))],
```

Comment:
This may cause bugs if isNameTagAlwaysVisible() is used, I recommend adding a check.